test/graph_rewrites/assign_clusters.cc

```diff
@@ -127,6 +136,15 @@ TEST(assign_clusters, cone) {
                 .Attr("_ngraph_static_inputs", std::vector<int32>{1})
                 .Finalize(&g, &node3));
 
+  // Add edges from SRC to node1 and node2
+  // Add edge from node3 to SINK
+  // The graph is disconnected without these edges
+  Node* source = g.source_node();
+  Node* sink = g.sink_node();
+  g.AddEdge(source, Graph::kControlSlot, node1, Graph::kControlSlot);
+  g.AddEdge(source, Graph::kControlSlot, node2, Graph::kControlSlot);
+  g.AddEdge(node3, Graph::kControlSlot, sink, Graph::kControlSlot);
```

Comment:
ah, so this is what you meant when u said 'current tests do not work'

Reply:
Yes :)